docs/api/datetime.md

```diff
@@ -273,6 +275,14 @@ Developers can provide their own buttons for advanced custom behavior.
 
 <CustomizingButtons />
 
+## Multiple Date Selection
+
+If the `multiple` property is set to `true`, multiple dates can be selected from the calendar picker at once. Clicking a selected date will deselect it.
+
+Note that this property is only supported when `presentation="date"`.
```

Comment:
~ Docusaurus does have callout components if we wanted to highlight this visually:

```html
:::note
This property is only supported when using `presentation="date"`.
:::
```